Classes/BackEnd/AbstractList.php

```diff
@@ -110,7 +111,7 @@ public function getEditIcon(int $recordUid, int $pageUid): string
         }
 
         $langEdit = \htmlspecialchars($this->getLanguageService()->getLL('edit'), ENT_QUOTES | ENT_HTML5);
-        $icon = '<img src="/' . ExtensionManagementUtility::siteRelPath('seminars') .
+        $icon = '<img src="' . PathUtility::getAbsoluteWebPath(ExtensionManagementUtility::extPath('seminars')) .
```

Comment:
We'll also need to have a leading slash before the absolute path as `getAbsoluteWebPath` does not add it. Hopefully, this will also fix the tests.

Ditto for the other places with the same change.